app/org/sagebionetworks/bridge/dynamodb/DynamoUploadSchema.java

```diff
@@ -151,6 +156,30 @@ public void setSchemaType(UploadSchemaType schemaType) {
         this.schemaType = schemaType;
     }
 
+    /** {@inheritDoc} */
+    @Override
+    public String getSurveyGuid() {
+        return surveyGuid;
+    }
+
+    /** @see #getSurveyGuid */
+    public void setSurveyGuid(String surveyGuid) {
+        this.surveyGuid = surveyGuid;
+    }
+
+    /** {@inheritDoc} */
+    @JsonSerialize(using = DateTimeToLongSerializer.class)
+    @Override
+    public Long getSurveyCreatedOn() {
+        return surveyCreatedOn;
+    }
+
+    /** @see #getSurveyCreatedOn */
+    @JsonDeserialize(using = DateTimeToPrimitiveLongDeserializer.class)
+    public void setSurveyCreatedOn(Long surveyCreatedOn) {
+        this.surveyCreatedOn = surveyCreatedOn;
+    }
```

Comment:
Can we add a flag to denote whether or not the schemas are auto-generated? My understanding is that the auto generated schemas would have surveyGuid/surveyCreatedOn, and so these would not need to be set in the schema editor UI.